### ISSUE TITLE
It is hard to customize formatting in IPython notebook

### ISSUE THREAD

**Issue by kmike** [TeamHG-Memex/eli5#69]
Currently in order to change formatting options in IPython notebook user has to do something like this:

```
from IPython.display import HTML
expl = explain_weights(clf, vec=fe, top=20)
HTML(format_as_html(expl, highlight_spaces=False, horizontal_layout=False))
```

It'd be nice to reduce it to a one-liner.

**Comment by lopuhin** [comment]
Yes, I also though about this. Maybe we can add a ``format_in_notebook`` or ``show_in_notebook`` function that will accept the same arguments as ``format_as_html``?

**Comment by kmike** [comment]
Do you propose this?

```py
expl = explain_weights(clf, vec=fe, top=20)
format_in_notebook(expl, highlight_spaces=False, horizontal_layout=False))
```

This is an improvement, but I think the main problem is that explanation/formatting separation makes sense internally, but it doesn't make much sense for the user. What do you think about e.g. ``eli5.show_weights`` and ``eli5.show_prediction`` which accept both explain_.. and format_.. keyword arguments, and return IPython-aware objects?

**Comment by lopuhin** [comment]
Yes, I like your idea better, that would be much more convenient!
